src/main/java/ru/investbook/InvestbookApplication.java

```diff
@@ -21,36 +21,34 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.web.servlet.context.ServletWebServerInitializedEvent;
 import org.springframework.cache.annotation.EnableCaching;
 import org.springframework.context.ApplicationContextException;
-import org.springframework.context.event.EventListener;
 import org.springframework.stereotype.Component;
 
+import java.io.IOException;
+
 @SpringBootApplication
 @Component
 @EnableCaching
 @RequiredArgsConstructor
 public class InvestbookApplication {
 
-    private final InvestbookProperties properties;
+    private static final LoadingPageServer loadingPageServer = new LoadingPageServer();
 
     public static void main(String[] args) {
+        try {
+            loadingPageServer.start();
+        } catch (IOException _) {}
```

Comment:
2 try blocks in the method don't look clean. Please, move 1-st try-catch into `start()` method and call `loadingPageServer.start()` inside 2-nd  try block. 

Also log warn message inside catch block, which is located in `start()` method. To do this use `@Slf4` annotation for `LoadingPageServer` class.